vpd-tool/include/tool_utils.hpp

```diff
@@ -75,6 +75,8 @@ inline types::DbusVariantType readDbusProperty(const std::string& i_serviceName,
  * @brief An API to print json data on stdout.
  *
  * @param[in] i_jsonData - JSON object.
+ *
+ * @throw std::runtime_error
```

Comment:
Don;t mix another fix with this commit.

Reply:
It's just a single line doxygen update for this API. No change has been made to the API itself.

Reply:
I do not agree with this change as in the other places where this is used may not have the exception check , so this changes a behavior of the function , so you will need to check all places where the function is used and accordingly flag the changes if any.

Reply:
To be clear, I did not modify the `printJson` method itself, I just updated its doxygen. The method throws `std::runtime_error` , its just that the doxygen did not document this, so I added the `@throw` line in the doxygen.